Source/WebCore/platform/graphics/freetype/FontCacheFreeType.cpp

```diff
@@ -367,7 +367,11 @@ std::unique_ptr<FontPlatformData> FontCache::createFontPlatformData(const FontDe
         return nullptr;
 
     FcChar8* fontConfigFamilyNameAfterMatching;
-    FcPatternGetString(resultPattern.get(), FC_FAMILY, 0, &fontConfigFamilyNameAfterMatching);
+
+    //When a font has two family names: use the second (more descriptive) one instead
```

Comment:
Not an expert with fontconfig, but please can you put a space at the start of your comment, it saves stylistic edits when changes are proposed upstream :)